src/EPR.CommonDataService.Data/Scripts/Views/create-large-producers-view.sql

```diff
@@ -7,144 +7,194 @@ DROP VIEW IF EXISTS [apps].[v_Large_Producers]
 GO
 
 CREATE VIEW [apps].[v_Large_Producers]
```

Comment:
Probably wants to be create or alter as afaik the pipeline doesn't nuke scripts first

Reply:
full code has a drop in front of this change, so create is valid